ignite/metrics/classification_report.py

```diff
@@ -0,0 +1,142 @@
+import json
+from typing import Callable, Collection, Dict, List, Optional, Sequence, Union
+
+import torch
+
+from ignite.metrics.fbeta import Fbeta
+from ignite.metrics.metric import Metric, reinit__is_reduced
+from ignite.metrics.metrics_lambda import MetricsLambda
+from ignite.metrics.precision import Precision
+from ignite.metrics.recall import Recall
+
+__all__ = ["ClassificationReport"]
+
+
+class ClassificationReport(MetricsLambda):
+    """Build a text report showing the main classification metrics. The report resembles in functionality
+        'https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html#sklearn.metrics.classification_report'
+        The underlying implementation doesn't use the sklearn function.
+
+        Args:
+            beta: weight of precision in harmonic mean
+            output_dict: If True, return output as dict, otherwise return a str
+            output_transform: a callable that is used to transform the
+                :class:`~ignite.engine.engine.Engine`'s ``process_function``'s output into the
+                form expected by the metric. This can be useful if, for example, you have a multi-output model and
+                you want to compute the metric with respect to one of the outputs.
+            device: optional device specification for internal storage.
+            labels: Optional list of label indices to include in the report
+            digits: Number of digits for formatting output floating point values, by default it is 5
```

Comment:
In scikit-learn this option is for the output as str and no truncation for the output as dict. I think either we should propose the same or remove `digits` argument... What do you think ?

Reply:
Good point I will remove the digits argument

Reply:
@Moh-Yakoub please hit "Resolve conversation" if the comment is/will be addressed and does not need more discussion. It is a convention that could be practical to follow when we contribute to OSS projects. Thanks.

Reply:
Please remove `digits` here

Reply:
@Moh-Yakoub ^^